### PR TITLE
[Docs] Correctly indent PyPI docs.

### LINKED ISSUE #688: Fix PyPI documentation formatting
## What

Fix the formatting for the package description

image

### PR DESCRIPTION
**Pull request by elliotcmorris**
Closes #688 

The indentation was causing the markdown formatting not to be appplied correctly.
test upload : https://test.pypi.org/project/openassetio/

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -50,22 +50,22 @@ Issues = "https://github.com/OpenAssetIO/OpenAssetIO/issues"
 # PyPI. Instead, boil down the first couple of sections to give a brief
 # overview.
 text = """
-    In modern creative pipelines, data is often managed by an authoritative system (Asset\
-    Management System, Digital Asset Manager, MAM, et. al).
+In modern creative pipelines, data is often managed by an authoritative system (Asset\
```
Comment (foundrytom) [inline pyproject.toml:53]: Just noticed - it needs a space before the `\` to avoid the words running together.

Comment (elliotcmorris) [reply pyproject.toml]: Spot. [f1bda67](https://github.com/OpenAssetIO/OpenAssetIO/pull/693/commits/f1bda6758da876a7ce291f3ec5ab6c7aedbd4f0b)

Comment (foundrytom) [reply pyproject.toml]: They all need updating, not just this one.

Comment (elliotcmorris) [reply pyproject.toml]: [44b6dca](https://github.com/OpenAssetIO/OpenAssetIO/pull/693/commits/44b6dca989533aa80287267bfbf02d53125bb95d)

**Review by feltech — APPROVED** [review]
Looks good to me. Would be nice to double-check the rendered output again before merging (the link in the description predates the whitespace fixes).  Other than that, for my part ready to squash and merge.

**Comment by elliotcmorris** [comment]
Final formatting check release : https://test.pypi.org/project/openassetio/

**Pull request merged**